test/controllers/StorageExplorerControllerSpec.scala

```diff
@@ -201,6 +202,31 @@ class StorageExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
     }
   }
 
+  "The file exploration controller" should {
+    "should return 404 if the requested node is not a bucket" in {
```

Comment:
why sometimes you start the sentence with "should" and sometimes not ?

Reply:
Human error. I will fix this.